localstack/services/awslambda/lambda_executors.py

```diff
@@ -54,6 +55,7 @@ class LambdaExecutor(object):
     def __init__(self):
         # keeps track of each function arn and the last time it was invoked
         self.function_invoke_times = {}
+        self.port = str(randint(10000, 12000))
```

Comment:
Can we use `get_free_tcp_port()` from `utils/common.py` here, instead of `randint`?